metadata.json

```diff
@@ -9,7 +9,7 @@
   "issues_url": "https://github.com/camptocamp/puppet-postfix/issues",
   "description": "Postfix Module for Puppet",
   "requirements": [
-    {"name":"puppet","version_requirement":">= 3.0.0 < 7.0.0" }
+    {"name":"puppet","version_requirement":">= 4.10.0 < 7.0.0" }
```

Comment:
The module hasn't actually supported puppet 3 since data types were added 2 years ago.